src/test/java/regres/get/users/single/user/GetSingleUserPositiveTest.java

```diff
@@ -14,18 +14,18 @@
 
 @RunWith(SerenityRunner.class)
 @WithTags({
-        @WithTag(type= "type", name = "GET")
+        @WithTag(type = "type", name = "GET")
 })
 public class GetSingleUserPositiveTest {
 
     @Steps
     private GetUserSteps steps;
 
-    ReusableMethods methods = new ReusableMethods();
+    private ReusableMethods methods = new ReusableMethods();
 
     @Test
     @WithTags({
-            @WithTag(type= "type", name = "Positive")
+            @WithTag(type = "type", name = "Positive")
     })
     @Title("GET a single user from the list based on id")
     public void getSingleUserFromList() {
```

Comment:
Same here, first you need to make sure the user exists and you create it in a before method